ansible/environments/local/group_vars/all

```diff
@@ -4,8 +4,10 @@ consul_conf_dir: /tmp/consul
 whisk_logs_dir: /tmp/wsklogs
 cli_conf_dir: /tmp/cli
 go_cli_conf_dir: /tmp/go-cli
+go_cli_nginx_dir: "{{ nginx_conf_dir }}/cli/go/download"
 docker_registry: ""
 docker_dns: ""
+docker_build_time_out: "240"
```

Comment:
`docker_build_time_out` is needed for all ansible environments? And why do we need a timeout here? It's not related to gradle?